stratosphere/ams_mitm/source/set_mitm/setsys_mitm_service.cpp

```diff
@@ -87,6 +88,87 @@ namespace ams::mitm::settings {
             return ResultSuccess();
         }
 
+        const char *g_external_bluetooth_db_location = "@Sdcard:/atmosphere/bluetooth_devices.db";
+
+        bool ExternalBluetoothDatabaseEnabled(void) {
```

Comment:
`IsExternalBluetoothDatabaseEnabled()`

Functions should never have explicit void parameter.